### PR TITLE
Fix regex for paging, add noWarn param for recursive main call to avo…

### PR DESCRIPTION
**Pull request by cbrown350**
…id warning

Regex doesn't account for additional div attributes that include quotes, breaking paging

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const URL = require('url');
 const UTIL = require('./util.js');
 const QS = require('querystring');
 
-const nextpagRegex = /<div class="(([^"]*branded\-page\-box[^"]*search-pager[^"]*)|([^"]*search-pager[^"]*branded\-page\-box[^"]*))">/; // eslint-disable-line no-useless-escape, max-len
+const nextpagRegex = /<div class="(([^"]*branded\-page\-box[^"]*search\-pager)|([^"]*search\-pager[^"]*branded\-page\-box)).*">/; // eslint-disable-line no-useless-escape, max-len
```
Comment (TimeForANinja) [inline lib/main.js:5]: why not remove it all together?
```suggestion
const nextpagRegex = /<div class="(([^"]*branded\-page\-box[^"]*search\-pager)|([^"]*search\-pager[^"]*branded\-page\-box))/; // eslint-disable-line no-useless-escape, max-len
```

Comment (cbrown350) [reply lib/main.js:5]: Seems reasonable assuming it's not later parsing data between brackets those.

Comment (cbrown350) [reply lib/main.js:5]: Seems reasonable assuming it's not later parsing data between brackets those.

**Comment by TimeForANinja** [comment]
thanks again for the pr
even thou it took me some time 😉

**Pull request merged**